Source/Core/DolphinQt/GameList/GameList.cpp

```diff
@@ -223,6 +228,20 @@ void GameList::MakeGridView()
           });
 }
 
+void GameList::ShowHeaderContextMenu(const QPoint& pos)
+{
+  const MenuBar* const menu_bar = MenuBar::GetMenuBar();
+  if (menu_bar)
```

Comment:
This could be inverted with an early return to remove one level of indent. Potentially the other one as well, but for just two lines....meh.

Reply:
I'll tweak it.